src/Adyen/Service/ResourceModel/Payment/TerminalCloudAPI.php

```diff
@@ -22,6 +36,6 @@ public function __construct($service, $asynchronous)
 		} else {
 			$this->endpoint = $service->getClient()->getConfig()->get('endpointTerminalCloud') . '/sync';
 		}
-		parent::__construct($service, $this->endpoint);
+		parent::__construct($service, $this->endpoint, $allowApplicationInfo = false, $this->allowApplicationInfoPOS);
```

Comment:
$allowApplicationInfo = false change this to $allowApplicationInfo.

Reply:
```suggestion
		parent::__construct($service, $this->endpoint, $this->allowApplicationInfo, $this->allowApplicationInfoPOS);
```